doc/catppuccin.txt

```diff
@@ -542,6 +542,10 @@ gitsigns.nvim>lua
     gitsigns = true
 <
 
+grug-far.nvim>lua
+    grug_far = true
```

Comment:
Seems that you've marked this integration as enabled by default, but it's actually not (in [lua/catppuccin/init.lua](https://github.com/catppuccin/nvim/blob/main/lua/catppuccin/init.lua)).

Reply:
Ok, cool. I had it as false before and the auto docs generation revised my commit to be true. Is there a way to fix that? I will go ahead and push back the `false` setting

Reply:
Hmm I'm not so familiar with the auto gen tool TBH, but anyway the doc is corrected now :+1: